client/src/API.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+export default {
+    getPokemonName: function () {
+        return axios.get('https://hackschool-fa20-test-server.herokuapp.com/api/pokemonName');
+    },
+
```

Comment:
store the heroku url in a variable (e.g. `serverUrl`) so you can do `url: serverUrl + '/pokemonName'`